app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+
+from app.errors import (
+    NotWearingMaskError,
+    VaccineError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    friend_with_no_mask = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(visitor=friend)
+        except (VaccineError):
```

Comment:
There is no need to place single exception in `()`, leave it as follows in the future
```suggestion
        except VaccineError:
```